Makefile

```diff
@@ -0,0 +1,18 @@
+SHELL := /bin/bash
+CWD := $(shell cd -P -- '$(shell dirname -- "$0")' && pwd -P)
+
+docker-images:
+	docker-compose build
+
+docker-npm-build:
+	rm -f .container_id
+	docker-compose run -d sphinx_rtd_theme build > .container_id
```

Comment:
nit: there is no need to do `rm -f` since you are redirecting the output with `>` in the next command. If it exists, it will overwrite it with the new output.

Reply:
haha yes, will remove it.... 'twas because `docker run` could write to a file - now I need to detach in order to get the ID because `docker-compose run` doesn't have the same option that `docker run` has.